lib/executor/ramping_vus.go

```diff
@@ -563,19 +563,20 @@ func (vlv RampingVUs) Run(parentCtx context.Context, out chan<- stats.SampleCont
 
 	activeVUsCount := new(int64)
 	vusFmt := pb.GetFixedLengthIntFormat(int64(maxVUs))
-	progresFn := func() (float64, []string) {
+	regularDurationStr := pb.GetFixedLengthDuration(regularDuration, regularDuration)
```

Comment:
Hhmm shouldn't `maxDuration` be used here instead?

Reply:
nope, we use the regular (i.e. user-specified) duration everywhere in the progressbars, I think - then after it expires, there gracefully stopping, but that might take no time at all, or the full 30s (or whatever was configured)